audit/audit.go

```diff
@@ -34,8 +36,23 @@ func init() {
 	if flag.Lookup("test.v") != nil {
 		return
 	}
+	var auditErrors []error
+
 	for _, pgConf := range conf.Backends.Postgres {
-		auditors = append(auditors, NewPostgresAuditor(pgConf.ConnStr))
+		auditDb, err := NewPostgresAuditor(pgConf.ConnStr)
+		if err != nil {
+			auditErrors = append(auditErrors, errors.Wrap(err, fmt.Sprintf("connectStr: %s, err", pgConf.ConnStr)))
+		}
+		auditors = append(auditors, auditDb)
+	}
+	if len(auditErrors) > 0 {
+		log.Errorf("connecting to audit database, defined in %s", configFile)
+		for i, err := range auditErrors {
+			if i == len(auditErrors)-1 {
```

Comment:
Sorry, missed this on my first pass. I think it'd be better to fail if _any_ of the audit backends fail, not just if the last one fails.

Other than this, lgtm. Could you give me permission to push to your branch, so I can vendor `pkg/errors`? There should a be checkbox for you to enable that within the PR.

Reply:
We prefer to vendor dependencies ourselves because it'd be very easy for someone to include malicious code in the vendoring commit, as we don't review the diffs for those.

Reply:
done  - "Allow edits from maintainers" is checked.